sql/_34_fig/cbrd_24252/cases/cbrd_24252_1.sql

```diff
@@ -37,7 +37,10 @@ from
     t_parent as p
     inner join t_child as c on c.parent_col_a = p.col_a
 where
-    c.col_b = -1;
+    c.col_b = -1
+order by
+    c.col_a,
+    c.col_b;
```

Comment:
```suggestion
order by
    c.col_a;
```
c.col_b = -1 조건이 있으므로 order by 절에 col_b는 필요가 없습니다. 
트레이스 결과에도 order by는 하나만 적용된 것을 볼 수 있습니다.
나머지 .sql에도 동일하게 적용 필요합니다.

Reply:
tc 작성 일관성을 위해 조회하는 모든 컬럼에 대하여 order by 를 추가하였습니다.

Reply:
리뷰 반영했습니다.